examples/python/example_with_zone.py

```diff
@@ -106,6 +106,18 @@ def write_data(data, connection):
                         default=get_default('print', False),
                         help='Print data (use --zone_id to set specific ZoneId '
                              'which will be used to fetch data)')
+    parser.add_argument('--ssl_mode', action='store_true',
```

Comment:
lte's use modern protocol names instead of legacy names. It's okay to leave `--ssl_mode`  because it's common name for mysql/postgresql parameter, but I suggest to use `tls_root_cert/tls_key/tls_cert` instead of `ssl_*`.